build/images/codegen/Dockerfile

```diff
@@ -46,7 +46,7 @@ RUN go install k8s.io/code-generator/cmd/client-gen@kubernetes-$K8S_VERSION && \
     go install k8s.io/kube-openapi/cmd/openapi-gen@$KUBEOPENAPI_VERSION && \
     go install k8s.io/code-generator/cmd/go-to-protobuf@kubernetes-$K8S_VERSION && \
     go install k8s.io/code-generator/cmd/go-to-protobuf/protoc-gen-gogo@kubernetes-$K8S_VERSION && \
-    go install github.com/golang/mock/mockgen@v1.4.4 && \
+    go install go.uber.org/mock/mockgen@v0.3.0 && \
```

Comment:
When you change this file, you actually need to rebuild the codegen image for the change to take effect (unfortunately this is a bit of a manual process).
To help out, I have added a commit to your PR, with the necessary changes for updating the tag of the codegen image.
You still need to do one thing: run `make codegen` and commit the changes (there will be changes to auto-generated files) as a 3rd commit to this PR.

Reply:
Sure thanks, I'm not sure if this is expected but it replaces `interface{} -> any ` which is essentially an alias and appends a period to the comments.

Reply:
That's expected. I got the same thing when testing locally. The `go.uber.org/mock` code generator has received some updates, hence these changes.